pkg/reconciler/v1alpha1/clusteringress/clusteringress.go

```diff
@@ -193,12 +203,28 @@ func (c *Reconciler) reconcile(ctx context.Context, ci *v1alpha1.ClusterIngress)
 		// when error reconciling VirtualService?
 		return err
 	}
+
 	// As underlying network programming (VirtualService now) is stateless,
 	// here we simply mark the ingress as ready if the VirtualService
 	// is successfully synced.
 	ci.Status.MarkNetworkConfigured()
 	ci.Status.MarkLoadBalancerReady(getLBStatus(gatewayServiceURLFromContext(ctx, ci)))
 	ci.Status.ObservedGeneration = ci.Generation
+
+	// TODO(zhiminx): copy certificate secrets into the target namespaces that Istio gateway
```

Comment:
Not sure I understand correctly, the comment, but perhaps like this?
```suggestion
	// TODO(zhiminx): copy certificate secrets into the target namespace where Istio gateway
```

Reply:
I rephrased the comment a bit. Please take a look.